taskfiles/build.yml

```diff
@@ -2,6 +2,79 @@ version: '3'
 
 tasks:
 
+  operator-image:
+    deps:
+    - alias # alias is used for backwards compat of entry points due to the lack of ln in distroless.
+    - operator
+    vars:
+      # Tags to tag this image as, defaulting to localhost/redpanda-operator:dev
+      TAGS: '{{ range $tag := (.TAGS | default (list "localhost/redpanda-operator:dev"))}} --tag {{ $tag }} {{end}}'
+      # Platforms to build the image for. NB: It's possible to build
+      # multi-platform images on machines without the containerd image store by
+      # using a docker-container buildkit builder. HOWEVER, it's not possible
+      # to `--load` the resultant image into docker if containerd image store
+      # is not enabled but you can `--push` it to a registry. Until our CI
+      # agents are updated to use the containerd image store, we have to dance
+      # around this limitation by modifying .CLI_ARGS and .PLATFORMS depending
+      # on the use of the built image.
+      # For local testing: 1 Platform (default should work) and `--load`. (build:image alias does this).
+      # For releases: Both platforms and `--push`.
+      PLATFORMS: '{{ join "," (.PLATFORMS | default (list (printf "linux/%s" ARCH)))}}'
+    cmds:
+    - docker buildx build
+          --platform {{ .PLATFORMS }}
+          --provenance false
+          --sbom false
+          --file operator/Dockerfile
+          --target=manager
+          {{ .TAGS }}
+          {{ .CLI_ARGS }}
+          .build
+
+  operator:
+    vars:
+      VERSION:
+        # TODO this should eventually be updated to 'operator/v*' to match go modules' tagging conventions.
+        sh: git describe --dirty --match 'v*'
```

Comment:
I think the `OPERATOR_VERSION` is used in `LD_FLAGS`, so maybe this `VERSION` variable can be removed?

Reply:
Good catch! Removed.